ar-module-explainer.md

```diff
@@ -45,8 +45,30 @@ This provides a session that behaves much like the immersive VR sessions describ
 
 The UA may choose to present the immersive AR session's content via any type of display, including dedicated XR hardware (for devices like HoloLens or Magic Leap) or 2D screens (for APIs like [ARKit](https://developer.apple.com/arkit/) and [ARCore](https://developers.google.com/ar/)). In all cases the session takes exclusive control of the display, hiding the rest of the page if necessary. On a phone screen, for example, this would mean that the session's content should be displayed in a mode that is distinct from standard page viewing, similar to the transition that happens when invoking the `requestFullscreen` API. The UA must also provide a way of exiting that mode and returning to the normal view of the page, at which point the immersive AR session must end.
 
+### Handling non-opaque displays
 
-#### Rays
+Some devices which support the WebXR Device API may use displays that are not fully opaque, or otherwise show your surrounding environment in some capacity. To determine how the display will blend rendered content with the real world, check the `XRSession`'s `environmentBlendMode` attribute. It may currently be one of three values, and more may be added in the future if new display technology necessitates it:
+
+  - `opaque`: The environment is not visible at all through this display. Transparent pixels in the `baseLayer` will appear black. This is the expected mode for most VR headsets. Alpha values will be ignored, with the compositor treating all alpha values as 1.0.
+  - `additive`: The environment is visible through the display and pixels in the `baseLayer` will be shown additively against it. Black pixels will appear fully transparent, and there is typically no way to make a pixel fully opaque. Alpha values will be ignored, with the compositor treating all alpha values as 1.0. This is the expected mode for devices like HoloLens or Magic Leap.
+  - `alpha-blend`: The environment is visible through the display and pixels in the `baseLayer` will be blended with it according to the alpha value of the pixel. Pixels with an alpha value of 1.0 will be fully opaque and pixels with an alpha value of 0.0 will be fully transparent. This is the expected mode for devices which use passthrough video to show the environment such as ARCore or ARKit enabled phones, as well as headsets that utilize passthrough video for AR like the Vive Pro.
```

Comment:
What does a device that optionally supports video passthrough (like a Vive Pro or Varjo XR-1) return for `XRSession.environmentBlendMode`?
1. Do we return `"opaque"` if you created an `"immersive-vr"` session (which _does not_ turn on the passthrough camera) and `"alpha-blend"` if you created an `"immersive-ar"` session (which _does_ turn on the passthrough camera)?
1. Do we return `"alpha-blend"` even for `"immersive-vr"` sessions, because you _could have_ turned on video passthrough by creating an `"immersive-ar"` session?
1. Do we return `"opaque"` even for `"immersive-ar"` sessions, because it is the "natural" state of the device?

Once a developer is already in an `XRSession`, I believe they will care most about how that session will actually behave.  My suggestion is that `XRSession.environmentBlendMode` be defined to represent the **"effective environment blend mode"** for the active session, informed both by the XR device that's in use and the `XRSessionMode` that the app selected.